packages/apidom-ast/src/index.ts

```diff
@@ -3,8 +3,10 @@ export { default as JsonNode } from './json/nodes/JsonNode.ts';
 export { default as JsonDocument } from './json/nodes/JsonDocument.ts';
 export { default as JsonObject } from './json/nodes/JsonObject.ts';
 export { default as JsonProperty } from './json/nodes/JsonProperty.ts';
+export type { JsonValue as JsonValueType } from './json/nodes/JsonProperty.ts';
```

Comment:
This needs to have a different name as we're already export `JsonValue` class.
Perhaps `JsonPropertyValue` would be better?
```suggestion
export type { JsonValue as JsonPropertyValue } from './json/nodes/JsonProperty.ts';
```

Reply:
Yes, go ahead with this.